doc/sphinx/Clusters_from_Scratch/verification.rst

```diff
@@ -150,8 +151,8 @@ point):
 
 .. NOTE::
 
-    Other operating systems may report startup errors in other locations,
-    for example ``/var/log/messages``.
+    Other operating systems may report startup errors in other locations
+    (e.g., ``/var/log/messages``).
```

Comment:
"for example" is preferred in documentation b/c it's easier for non-native speakers. Not that I always remember that ...

Reply:
Ack, done